compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1948,3 +1948,9 @@ error.int.range.overflow=\
 
 error.unsupported.usage.of.default.values.for.key.field.in.table.member=\
   using default values of record fields for table key fields is not yet supported
+
+error.worker.send.action.not.allowed.in.lock.statement=\
+  using send action not allowed in lock statement
```

Comment:
This is good, Shall we add why as well? i.e.,
`using send action within the lock statement is not allowed to prevent possible deadlocks`. This is deviating from our existing message form. But giving a reason is good IMO. Do the same for others.

Reply:
Updated in 71cf6d05b3840513a959f848b8cf4d389670a66b